### DISCUSSION TITLE
The snap sector states are Available always in some days

### DISCUSSION THREAD

**Discussion by apojii** [filecoin-project/lotus#11163] [Storage Provider]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus fvm/fevm - Lotus FVM and FEVM interactions
- [X] lotus miner/worker - sealing
- [X] lotus miner - proving(WindowPoSt/WinningPoSt)
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
1.23.0
```

### Repro Steps

1. Run 'lotus-miner sectors snap-up 7953'
2. Do 'lotus-miner sectors list'
3. The sector of 7953 is Available State.
...

### Describe the Bug

The snap sector states are Available always in some days, not turns to SnapDealsWaitDeals.
I want to upgrade the CC sectors, then I run the command snap-up for this:
root@master:~# lotus-miner sectors snap-up 7953

But the sectors state keeps in Available, not turns to the state of SnapDealsWaitDeals, wait for some days are the same.
root@master:~# lotus-miner sectors list | grep Available
4790 Available YES YES 4158887 (in 51 weeks 3 days) CC
7950 Available YES YES 4656431 (in 1 year 24 weeks) CC
7951 Available YES YES 4656472 (in 1 year 24 weeks) CC
7952 Available YES YES 4658536 (in 1 year 24 weeks) CC
7953 Available YES YES 4655707 (in 1 year 24 weeks) CC

The miner config is default value.
What can I do to change this?

### Logging Information

```text
No error log.
```

**Comment by TippyFlitsUK** [comment]
Have you checked that the expiration of the sector is adequate to host the deal?

**Reply by apojii** [reply]
Yes, most of the sectors' expiration are 1 year 24 weeks, that are adequate to host the deal.
